VWOSdk/Internal/Core/SegmentEvaluator.cs

```diff
@@ -83,7 +98,48 @@ public dynamic getTypeCastedFeatureValue(dynamic value, string variableType)
                 return null;
             }
         }
+        public bool IsValidJson(string input)
+        {
+            input = input.Trim();
+            if ((input.StartsWith("{") && input.EndsWith("}")) || //For object
+                (input.StartsWith("[") && input.EndsWith("]"))) //For array
+            {
+                try
+                {
+                    //parse the input into a JObject
+                    var jObject = JObject.Parse(input);
+
+                    foreach (var jo in jObject)
+                    {
+                        string name = jo.Key;
+                        JToken value = jo.Value;
+
+                        //if the element has a missing value, it will be Undefined - this is invalid
+                        if (value.Type == JTokenType.Undefined)
+                        {
+                            return false;
+                        }
+                    }
+                }
+                catch (JsonReaderException jex)
+                {
+                    //Exception in parsing json
+                    Console.WriteLine(jex.Message);
+                    return false;
+                }
+                catch (Exception ex) //some other exception
+                {
+                    Console.WriteLine(ex.ToString());
```

Comment:
Have we used Console commands anywhere else in the code if not please remove from here as well.